frontend/app/forms/MemberForm.scala

```diff
@@ -109,7 +135,7 @@ object MemberForm {
     )(StaffJoinForm.apply)(StaffJoinForm.unapply)
   )
 
-  val paidMemberJoinForm: Form[PaidMemberJoinForm] = Form(
+  def paidMemberJoinForm: Form[PaidMemberJoinForm] = Form(
```

Comment:
Well spotted once again! This was a left over from some changes I eventually haven't committed!